coop_account/migrations/12.0.1.1.0/pre-migrate.py

```diff
@@ -0,0 +1,13 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    selected_journals = env['account.journal'].search([
+        ('name', 'not like', '%Chèques%'),
+        '|',
+        ('name', 'like', 'CCOOP - compte courant'),
+        ('name', 'ilike', '%cep%'),
+    ])
+    selected_journals.write({'export_wrong_reconciliation': True})
```

Comment:
The file name should be `post-migrate.py`.

`pre` scripts are run before your module update
`post` are run after your module update.

You need it to run **after** because you want to write on a field that your module is adding.
If you run it before, it will fail because the field doesn't exist

Reply:
@ivantodorovich : done